transport/tchannel/options.go

```diff
@@ -220,3 +221,10 @@ func WithNativeTChannelMethods(nativeMethods NativeTChannelMethods) TransportOpt
 		option.nativeTChannelMethods = nativeMethods
 	}
 }
+
+// WithDisableRpcServiceHeader disables adding the $rpc$-service header
+func WithDisableRpcServiceHeader() TransportOption {
```

Comment:
With is not 100% necessary here. `DisableRpcServiceHeader()` is a better name

Reply:
Done. thanks